packages/core/src/modules/attendance/attendance-pool-repository.ts

```diff
@@ -0,0 +1,109 @@
+import { type Database } from "@dotkomonline/db"
+import { type DeleteResult } from "@dotkomonline/db/utils"
+import {
+  AttendancePool,
+  AttendancePoolSchema,
+  type AttendanceId,
+  type AttendancePoolId,
+  type AttendancePoolWrite,
+} from "@dotkomonline/types"
+import { Selectable, type Kysely } from "kysely"
+import { prepareJsonInsert } from "../../utils/db-utils"
+import { z } from "zod"
+
+type DatabasePool = Selectable<Database["attendancePool"]> & { numAttendees: number }
```

Comment:
We should probably make it consistent where we use Selectable/other types. I'll clean up that in a separte PR though!